.github/workflows/python-publish.yml

```diff
@@ -26,3 +26,26 @@ jobs:
 
     - name: Publish release to pypi
       run: poetry publish --build
+
+    - name: Get Upload URL
+      id: create_release
+      shell: bash
+      env:
+        access_token: ${{ secrets.GITHUB_TOKEN }}
+        tag_act: ${{ github.ref }}
+      run: |
+          ref_tag=$(python -c "print('${tag_act}'.split('/')[-1])")
```

Comment:
What is ${tag_act}?

Reply:
That would have some value on which tag it runs for example `refs/origin/v0.1.0`.